app/presenters/manual_presenter.rb

```diff
@@ -16,7 +16,7 @@ def summary
   end
 
   def body
-    GovspeakToHTMLRenderer.create.call(manual).attributes.fetch(:body)
+    GovspeakHtmlConverter.new.call(manual.body)
```

Comment:
I think this change makes the `require "govspeak_to_html_renderer"` statement at the top of the file redundant.